.eslintrc.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    env: {
+        browser: true,
+        commonjs: true,
+        es6: true,
+        node: true,
+    },
+    extends: 'eslint:recommended',
```

Comment:
why not extend from `eventbrite`? https://github.com/eventbrite/javascript/tree/master/packages/eslint-config-eventbrite

Reply:
yeah! will do!